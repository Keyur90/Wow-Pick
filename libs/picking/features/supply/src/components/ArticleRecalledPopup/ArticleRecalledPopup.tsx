import React, { useState } from 'react';
import { PopUpModal } from '@rf2/ui';

interface ArticleRecalledPopupProps {
  isOpen: boolean;
  onClose: () => void;
}

const ArticleRecalledPopup: React.FC<ArticleRecalledPopupProps> = ({ isOpen, onClose }) => {
  return (
    <PopUpModal
      isOpen={isOpen}
      onOkClick={onClose}
      bodyText="Product recalled/withdrawn, not for sale"
      headerText="Order"
      closeOrOkText="Close"
    />
  );
};

export { ArticleRecalledPopup };
