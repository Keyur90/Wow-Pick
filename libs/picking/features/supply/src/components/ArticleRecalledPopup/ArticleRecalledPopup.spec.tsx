import React from 'react';
import { render } from '@testing-library/react';
import { ArticleRecalledPopup } from './ArticleRecalledPopup';

test('renders ArticleRecalledPopup', () => {
  const { baseElement } = render(<ArticleRecalledPopup isOpen={true} onClose={null} />);

  expect(baseElement).toMatchSnapshot();
});
