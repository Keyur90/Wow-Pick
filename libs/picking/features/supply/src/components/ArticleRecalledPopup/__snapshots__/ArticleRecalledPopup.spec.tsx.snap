// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ArticleRecalledPopup 1`] = `
<body
  style="padding-right: 0px; overflow: hidden;"
>
  <div
    aria-hidden="true"
  />
  <div
    class="MuiDialog-root"
    role="presentation"
    style="position: fixed; z-index: 1300; right: 0px; bottom: 0px; top: 0px; left: 0px;"
  >
    <div
      aria-hidden="true"
      class="MuiBackdrop-root"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    />
    <div
      data-test="sentinelStart"
      tabindex="0"
    />
    <div
      class="MuiDialog-container MuiDialog-scrollPaper"
      role="none presentation"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      tabindex="-1"
    >
      <div
        class="MuiPaper-root MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm MuiDialog-paperFullWidth MuiPaper-elevation24 MuiPaper-rounded"
        role="dialog"
      >
        <div
          class="MuiDialogContent-root makeStyles-root-1"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3"
          >
            Order
          </h3>
          <p
            class="MuiTypography-root MuiTypography-body1"
          >
            Product recalled/withdrawn, not for sale
          </p>
        </div>
        <div
          class="MuiDialogActions-root makeStyles-root-2 MuiDialogActions-spacing"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              Close
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      data-test="sentinelEnd"
      tabindex="0"
    />
  </div>
</body>
`;
